internal/commands/config_pull_policy.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	pubcfg "github.com/buildpacks/pack/config"
```

Comment:
I played around with moving the `PullPolicy` type to places other than this public config, but none of those places ended up necessarily making more sense than keeping it where it is.